src/components/Request/FormFields/CloseDateLinkedinPost.tsx

```diff
@@ -0,0 +1,60 @@
+import { DatePicker } from "@fluentui/react";
+import { Label, Text } from "@fluentui/react-components";
+import { CalendarIcon } from "@fluentui/react-icons-mdl2";
+import { Controller } from "react-hook-form";
+import "components/Request/Request.css";
+import { FormField } from "components/Request/NewRequestForm";
+
+const onFormatDate = (date?: Date): string => {
+  return !date
+    ? ""
+    : date.getMonth() + 1 + "/" + date.getDate() + "/" + date.getFullYear();
+};
+
+const CloseDateLinkedinPost = ({ name, form }: FormField) => {
+  const today = new Date(Date.now());
+
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        id={name + "Id"}
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <CalendarIcon className="requestFieldIcon" />
+        Close date for LinkedIn Job Posting
+      </Label>
+      <Controller
```

Comment:
Looks like BAC-58 AC4 indicates there really is no date selection available -- just a note that says it will be active for 30 days.